lib/user.js

```diff
@@ -78,28 +102,40 @@ const insertNewUser = async function (claims) {
 
 const ensureFreshUserData = async function (person, claims) {
   const properties = [
-    { predicate: 'foaf:firstName', oldValue: person.firstName, newValue: claims[AUTH_FIRST_NAME_CLAIM] },
-    { predicate: 'foaf:familyName', oldValue: person.familyName, newValue: claims[AUTH_FAMILY_NAME_CLAIM] }
+    {
+      predicate: 'http://xmlns.com/foaf/0.1/firstName',
+      oldValue: person.firstName,
+      newValue: claims[AUTH_FIRST_NAME_CLAIM],
+      escapeMethod: sparqlEscapeString
+    },
+    {
+      predicate: 'http://xmlns.com/foaf/0.1/familyName',
+      oldValue: person.familyName,
+      newValue: claims[AUTH_FAMILY_NAME_CLAIM],
+      escapeMethod: sparqlEscapeString
+    },
+    {
+      predicate: 'http://xmlns.com/foaf/0.1/mbox',
+      oldValue: person.email,
+      newValue: claims[AUTH_EMAIL_CLAIM] ? emailToUri(parseEmailFromClaim(claims[AUTH_EMAIL_CLAIM])) : undefined,
+      escapeMethod: sparqlEscapeUri
+    }
   ];
 
-  for (const { predicate, oldValue, newValue } of properties) {
+  for (const { predicate, oldValue, newValue, escapeMethod } of properties) {
     if (oldValue != newValue) {
       await update(`
-        PREFIX foaf: <http://xmlns.com/foaf/0.1/>
-
         DELETE WHERE {
           GRAPH <${USERS_GRAPH}> {
-            ${sparqlEscapeUri(person.uri)} ${sparqlEscapeUri(predicate)} ?value .
+            ${sparqlEscapeUri(person.uri)} ${predicate} ?value .
```

Comment:
Now we do actually need to keep the `sparqlEscapeUri` here because the predicate is now the whole URI and not a prefixed statement.

Reply:
oops, misread this during the merge. Will fix